MIGRATION_PLAN.md

```diff
@@ -21,4 +21,4 @@ Migrate all usage to `java.net.http.HttpClient` (available since Java 11).
 
 ## Progress
 
-Currently targeting development-java-11 branch as requested.
\ No newline at end of file
+Currently targeting development-java-11 branch as requested.
```

Comment:
This plan is for a different issue and needs moved to the plans folder.  The development-java-11 branch has been merged to development.  Java modernization changes should be made on development going forward.  The JDK target is now Java 17.